evaluator/evaluator.py

```diff
@@ -338,29 +340,42 @@ def run(self):
         # pylint: disable=too-many-branches
         def process_message(message):
             """Message procession logic"""
+            try:
+                msg_dict = json.loads(message.value.decode('utf-8'))
+                FailedCache.process_failed_cache(FailedCache.upload_cache, executor, self.process_upload_or_re_evaluate, loop)
+            except json.decoder.JSONDecodeError:
+                MESSAGE_PARSE_ERROR.inc()
+                LOGGER.exception("Unable to parse message: ")
+                return
             if message.topic in kafka_evaluator_topic:
-                try:
-                    msg_dict = json.loads(message.value.decode('utf-8'))
-                    FailedCache.process_failed_cache(FailedCache.upload_cache, executor, self.process_upload_or_re_evaluate, loop)
-                except json.decoder.JSONDecodeError:
-                    MESSAGE_PARSE_ERROR.inc()
-                    LOGGER.exception("Unable to parse message: ")
-                    return
                 if 'type' not in msg_dict:
                     LOGGER.error("Received message is missing type field: %s", msg_dict)
+                    send_msg_to_payload_tracker(PAYLOAD_TRACKER_PRODUCER, msg_dict, 'error',
```

Comment:
maybe delete this message, it may not be related to payload processing, some attributes used in payload tracker msg can be missing in msg_dict